op-program/client/boot/boot_interop.go

```diff
@@ -37,6 +39,7 @@ type OracleConfigSource struct {
 
 	l2ChainConfigs map[eth.ChainID]*params.ChainConfig
 	rollupConfigs  map[eth.ChainID]*rollup.Config
+	depset         *depset.StaticConfigDependencySet
```

Comment:
nit: We probably should use the `depset.DependencySet` interface here.

Reply:
Indeed. Now using the interface everywhere.